common/artifacts/909_wc_artifacts_library_librams.txt

```diff
@@ -379,21 +358,19 @@ libram_of_guardianship_01 = {
 	stacking = no
 	indestructible = yes
 
-	martial = 1
-	health = 0.1
-	combat_rating = 1
-	
-	monthly_character_prestige = 0.04
-	monthly_character_piety = 0.08
-	
-	light_group_opinion = 1
+	martial = 2
+	health = 0.325
```

Comment:
Thousandths don't work in ck2
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/76130699-c9d8c980-601c-11ea-9a04-2e1bf8c8e6be.png)

</details>

Reply:
fixed